### ISSUE TITLE
Explain how to "ask for another round of review" via rustbot labels

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rustc-dev-guide#1123]
New contributors currently have no good way of knowing this magic incantation
```
@rustbot label -S-waiting-on-author +S-waiting-on-review
```
It should probably be mentioned somewhere on https://rustc-dev-guide.rust-lang.org/contributing.html.

See https://github.com/rust-lang/rust/pull/82703 for a recent case where this came up (and I've certainly seen other instances of this before).

**Comment by iago-lito** [comment]
Well, now I have found it [there](https://rustc-dev-guide.rust-lang.org/contributing.html#opening-a-pr), pointing [there](https://rustc-dev-guide.rust-lang.org/rustbot.html#issue-relabeling). The problem was that I was so busy figuring out the `How to Build and Run the Compiler` and the `compiler testing framework` back then that I did not make my way down to this section.

But now that I am mostly done with implementation, maybe I should've simply come back to this book earlier?
